feature/repair/src/main/java/com/iteneum/repair/data/RepairViewModel.kt

```diff
@@ -0,0 +1,81 @@
+package com.iteneum.repair.data
+
+import android.net.Uri
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.core.text.isDigitsOnly
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.iteneum.RepairModel
+import com.iteneum.network.DataState
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+/**
+ * File that contains [RepairViewModel] that works as ViewModel for the Repair View.
+ * It will handle the data work & process when loading & be done with the view.
+ *
+ * @author Jose Miguel Garcia Reyes
+ */
+
+
+@HiltViewModel
+class RepairViewModel @Inject constructor(): ViewModel() {
+    private val _repairModelMutable : MutableState<RepairModel> = mutableStateOf(RepairModel())
```

Comment:
if we are using a mutbleStateOf we cand change the implementation like

var  myData: RepairModel by mutableStateOf(RepairModel)
        private set

Reply:
Ok done! Thanks for the explanation in person as well